src/holofuel/pages/CreateRequest/CreateRequest.test.js

```diff
@@ -227,6 +229,57 @@ describe('CreateRequest', () => {
     expect(within(getByTestId('counterparty-nickname')).getByText('Loading')).toBeInTheDocument()
   })
 
+  it('renders a clickable list of recent counterparties', async () => {
```

Comment:
obv the duplication between this and `CreateOffer.test` is pretty egregious, but I think this is the best solution til we merge them. Definitely feeling ready to merge them :D